src/main.rs

```diff
@@ -32,6 +32,7 @@ fn run_blocking() -> anyhow::Result<()> {
         .on_thread_stop(|| {
             TRACING_GUARD.take();
         })
+        .enable_all()
```

Comment:
Why do we need this?

Reply:
It enables http and file IO in the tokio runtime

I can't figure out why it's needed for reflection and it wasn't needed earlier..